augur/postprocess.py

```diff
@@ -0,0 +1,195 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.collections import EllipseCollection
+import astropy.table
+from scipy.stats import norm, chi2
+import os
+
+
+def postprocess(config):
```

Comment:
This file is not new to this branch, right? It is already in master.

Reply:
No, this is in master already. I have to rebase.